backend/compilers/download.py

```diff
@@ -161,6 +161,43 @@ def download_zip(
             f.extract(member=file, path=dest_path)
 
 
+def download_codewarrior():
+    download_zip(
+        url="https://github.com/ChrisNonyminus/sims1_mac_decomp/files/8748510/compilers.zip",
+        dl_name="codewarrior_compilers.zip",
+        dest_name="codewarrior",
+        create_subdir=True,
+    )
+    compiler_dir = COMPILERS_DIR / "codewarrior" / "compilers"
+    lowercase_lmgr = compiler_dir / "Pro5" / "lmgr326b.dll"
+    if lowercase_lmgr.exists():
+        shutil.move(lowercase_lmgr, compiler_dir / "Pro5" / "LMGR326B.dll")
+
+    lowercase_lmgr = compiler_dir / "Pro5" / "lmgr8c.dll"
+    if lowercase_lmgr.exists():
+        shutil.move(lowercase_lmgr, compiler_dir / "Pro5" / "LMGR8C.dll")
+
+    # Set +x to allow WSL without wine
+    exe_path = compiler_dir / "Pro5" / "MWCPPC.exe"
+    exe_path.chmod(exe_path.stat().st_mode | stat.S_IEXEC)
+
+    compiler_dir = COMPILERS_DIR / "codewarrior" / "compilers"
+    lowercase_lmgr = compiler_dir / "Pro6" / "lmgr326b.dll"
+    if lowercase_lmgr.exists():
+        shutil.move(lowercase_lmgr, compiler_dir / "Pro6" / "LMGR326B.dll")
+
+    lowercase_lmgr = compiler_dir / "Pro6" / "lmgr8c.dll"
+    if lowercase_lmgr.exists():
+        shutil.move(lowercase_lmgr, compiler_dir / "Pro6" / "LMGR8C.dll")
+
+    # Set +x to allow WSL without wine
+    exe_path = compiler_dir / "Pro6" / "MWCPPC.exe"
+    exe_path.chmod(exe_path.stat().st_mode | stat.S_IEXEC)
```

Comment:
I'd prefer we do this in a loop like `for ver in ["Pro5", "Pro6"] rather than duplicating this code twice